### PR TITLE
.eb file for cairo-1.12.18-goolf-1.7.20

### PR DESCRIPTION
**Pull request by npklein**
depends on #1710, #1711, #1714, #1715

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/c/cairo/cairo-1.12.18-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = "cairo"
+version = '1.12.18'
+
+homepage = 'http://cairographics.org'
+description = """Cairo is a 2D graphics library with support for multiple output devices.
+Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
+PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB"""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/c/cairo/cairo-1.12.18-goolf-1.7.20.eb]: if you can do 1 leading space here, that's nice
(sry to nag about it, yet I've been reviewed in the same way!).

File: easybuild/easyconfigs/c/cairo/cairo-1.12.18-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('libpng', '1.6.12'),
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/c/cairo/cairo-1.12.18-goolf-1.7.20.eb]: ah, yes, if  the above could be nicely aligned with common_deps #1689...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/65c25dda7709989973ef for a full test report.

**Comment by boegel** [comment]
Going in, thanks @npklein!

**Pull request merged**